de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/RunnerNodeModel.java

```diff
@@ -73,7 +73,7 @@
   private final RunnerNodeInternalSettings internalSettings = new RunnerNodeInternalSettings();
 
   private RunnerNodeSettings nodeSettings = new RunnerNodeSettings();
-
+  private FskPortObject fskObj = null;
```

Comment:
For keeping port objects I would implement the PortObjectHolder interface. This requires the getInternalPortObjects() and setInternalPortObjects() methods.